src/App.tsx

```diff
@@ -15,36 +17,93 @@
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  length = 'length',
+  alph = 'alph',
+  default = '',
+}
+
+export const App = () => {
+  const [isReversed, setIsReversed] = useState(false);
+  const [sortType, setSortType] = useState(SortType.default);
+
+  const reverseList = () => {
+    if (isReversed) {
+      setIsReversed(false);
+    } else {
+      setIsReversed(true);
+    }
```

Comment:
The `reverseList` function can be simplified by using `setIsReversed(!isReversed)`. This will toggle the boolean value without needing an if-else statement.